### PR TITLE
Created doc: What is Nebula Graph

### PR DESCRIPTION
**Pull request by randomJoe211**

### CODE REVIEW

File: docs_2.0/manual_EN/2.introduction/1.what-is-nebula-graph.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+# What is Nebula Graph
+
+Nebula Graph is an open-source, distributed, easily scalable, and native graph database. It is capable of hosting graphs with billions of vertices and trillions of edges, and serve queries with millisecond-latency.
+
+## What is a graph database
+
+A graph database, such as Nebula Graph, is a storage engine that specializes in storing and retrieving vast networks of information. It efficiently stores data as vertices (nodes) and edges (relationships) and allows high-performance retrieval and querying of those structures. Properties can be added to both vertices and edges. Vertices can have zero to multiple tags, and edges are always directed and typed.
```
Comment (whitewum) [inline docs_2.0/manual_EN/2.introduction/1.what-is-nebula-graph.md:7]: not a storage engine but a database

Comment (whitewum) [inline docs_2.0/manual_EN/2.introduction/1.what-is-nebula-graph.md:7]: Properties can be added --attached -- to both vertices and edges. Vertices can have zero -- at least one -- to multiple tags, and edges are always directed and typed.

File: docs_2.0/manual_EN/2.introduction/1.what-is-nebula-graph.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+[TODO: Add a png of a simple graph.]
+
+Graph databases are well suited for storing most kinds of data models abstracted from reality. In almost all fields, there are certain things connected to other things. In most other modeling approaches, the relationships between things are reduced to a single link without identity and attributes. Nebula Graph, as a typical native graph database, allows you to keep the rich relationships that originated from reality equally well-represented in the database without resorting to also modeling the relationships as entities.
```
Comment (whitewum) [inline docs_2.0/manual_EN/2.introduction/1.what-is-nebula-graph.md:11]: why `a single link without ... attributes`?

File: docs_2.0/manual_EN/2.introduction/1.what-is-nebula-graph.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+You can easily model the connected data into Nebula Graph for your business without forcing them into a structure such as a relational table, and properties can be added, updated, and deleted freely. For more information, see [Data modeling](TODO) [TODO] and [Nebula Graph CRUD](../3.quick-start/4.nebula-graph-crud.md).
+
+### Outstanding performance
+
+Written in C++ and born for graph, Nebula Graph handles graph queries in milliseconds. Among most databases, Nebula Graph shows superior performance in providing graph data services. The larger the data size, the greater the superiority of Nebula Graph. For more information, see [Nebula Graph benchmarking](https://discuss.nebula-graph.io/t/nebula-graph-1-0-benchmark-report/581).
+
+### High popularity
+
+Nebula Graph is being used by tech leaders such as Tencent, Vivo, Meituan, JD Digits., and a lot more.
```
Comment (whitewum) [inline docs_2.0/manual_EN/2.introduction/1.what-is-nebula-graph.md:25]: must link to somewhere

File: docs_2.0/manual_EN/2.introduction/1.what-is-nebula-graph.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+### Open-source
```
Comment (whitewum) [inline docs_2.0/manual_EN/2.introduction/1.what-is-nebula-graph.md:27]: make open-source as the first

File: docs_2.0/manual_EN/2.introduction/1.what-is-nebula-graph.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+
+### High scalability
+
+Nebula Graph is designed in a shared-nothing architecture and supports scaling in and out without interrupting the database service.
+
+### Diversified ecosystem
+
+[TODO]
+
+### SQL-like query language
```
Comment (whitewum) [inline docs_2.0/manual_EN/2.introduction/1.what-is-nebula-graph.md:47]: openCypher Compatability. not SQL-like.

Comment (whitewum) [inline docs_2.0/manual_EN/2.introduction/1.what-is-nebula-graph.md:47]: near by Developer friendly

File: docs_2.0/manual_EN/2.introduction/1.what-is-nebula-graph.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+
+A graph database, such as Nebula Graph, is a storage engine that specializes in storing and retrieving vast networks of information. It efficiently stores data as vertices (nodes) and edges (relationships) and allows high-performance retrieval and querying of those structures. Properties can be added to both vertices and edges. Vertices can have zero to multiple tags, and edges are always directed and typed.
+
+[TODO: Add a png of a simple graph.]
+
+Graph databases are well suited for storing most kinds of data models abstracted from reality. In almost all fields, there are certain things connected to other things. In most other modeling approaches, the relationships between things are reduced to a single link without identity and attributes. Nebula Graph, as a typical native graph database, allows you to keep the rich relationships that originated from reality equally well-represented in the database without resorting to also modeling the relationships as entities.
+
+## Benefits of Nebula Graph
+
+### Easy data modeling and high flexibility
```
Comment (whitewum) [inline docs_2.0/manual_EN/2.introduction/1.what-is-nebula-graph.md:15]: consider re-ordering these benefits in logical order.

**Review by whitewum — CHANGES_REQUESTED** [review]
some errors.

File: docs_2.0/manual_EN/2.introduction/1.what-is-nebula-graph.md (hunk 2 above)
Comment (oldLady344) [inline docs_2.0/manual_EN/2.introduction/1.what-is-nebula-graph.md:11]: "Nebula Graph, as a typical native graph database, allows you to keep the rich relationships that originated from reality equally well-represented in the database without resorting to also modeling the relationships as entities." ==> Nebula Graph, as a typical native graph database, allows you to keep the rich relationships that 【are？】originated from reality equally well-represented in the database without resorting to 【also？】 modeling the relationships as entities.

Suggestion: Always avoid writing long and complex sentences.

File: docs_2.0/manual_EN/2.introduction/1.what-is-nebula-graph.md (hunk 3 above)
Comment (oldLady344) [inline docs_2.0/manual_EN/2.introduction/1.what-is-nebula-graph.md:25]: remove ", and a lot more."

File: docs_2.0/manual_EN/2.introduction/1.what-is-nebula-graph.md [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+
+Written in C++ and born for graph, Nebula Graph handles graph queries in milliseconds. Among most databases, Nebula Graph shows superior performance in providing graph data services. The larger the data size, the greater the superiority of Nebula Graph. For more information, see [Nebula Graph benchmarking](https://discuss.nebula-graph.io/t/nebula-graph-1-0-benchmark-report/581).
+
+### High popularity
+
+Nebula Graph is being used by tech leaders such as Tencent, Vivo, Meituan, JD Digits., and a lot more.
+
+### Open-source
+
+Nebula Graph is open under the Apache 2.0 and the Commons Clause 1.0 license. More and more people such as database developers, data scientists, security experts, algorithm engineers are participating in the designing and development of Nebula Graph. To join the opening of source code and ideas, surf the [Nebula Graph Github page](https://github.com/vesoft-inc/nebula-graph).
```
Comment (oldLady344) [inline docs_2.0/manual_EN/2.introduction/1.what-is-nebula-graph.md:29]: licenses.

Comment (oldLady344) [inline docs_2.0/manual_EN/2.introduction/1.what-is-nebula-graph.md:29]: , and algorithm engineers

Comment (oldLady344) [inline docs_2.0/manual_EN/2.introduction/1.what-is-nebula-graph.md:29]: GitHub

File: docs_2.0/manual_EN/2.introduction/1.what-is-nebula-graph.md [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+
+### Developer friendly
+
+Nebula Graph supports clients in popular programming languages like Java, Python, C++, and Go, and more are being developed. For more information, see [Nebula Graph clients](TODO) [TODO].
+
+### Reliable access control
+
+Nebula Graph supports strict role-based ACL and external authentication servers such as LDAP to enhance data security. For more information, see [Authentication and authorization](TODO) [TODO].
```
Comment (oldLady344) [inline docs_2.0/manual_EN/2.introduction/1.what-is-nebula-graph.md:37]: spell out LDAP in parentheses?

File: docs_2.0/manual_EN/2.introduction/1.what-is-nebula-graph.md [hunk 9]
```diff
@@ -0,0 +1,69 @@
...
+
+The native Nebula Graph Query Language (nGQL), is a declarative, textual query language like SQL, but for graphs. It is easy to understand and easy to use. Starting from Nebula Graph 2.0.0-alpha, nGQL is on the way to be fully compatible with openCypher. For more information, see [nGQL guide](https://github.com/vesoft-inc/nebula-docs/tree/master/docs_2.0/manual_EN/4.ngql-guide).
+
+## Use cases
+
+To spare the time spent on pushing the kinds of data mentioned in this section into relational databases and on bothering for join queries, use Nebula Graph.
+
+### Fraud detection
+
+Financial institutions have to traverse countless transactions to piece together potential crimes, and understand how combinations of transactions and devices might be related to a single fraud scheme. This kind of scenario can be modeled in graphs, and with the help of Nebula Graph, fraud rings and other sophisticated scams can be easily detected.
```
Comment (oldLady344) [inline docs_2.0/manual_EN/2.introduction/1.what-is-nebula-graph.md:57]: "potential crimes, and understand" remove the comma?

File: docs_2.0/manual_EN/2.introduction/1.what-is-nebula-graph.md [hunk 10]
```diff
@@ -0,0 +1,69 @@
...
+
+### Real-time recommendation
+
+Nebula Graph offers the ability to instantly process the real-time information a visitor and make accurate recommendations on articles, videos, products, and services.
```
Comment (oldLady344) [inline docs_2.0/manual_EN/2.introduction/1.what-is-nebula-graph.md:61]: "the real-time information a visitor and make" any word missing?

File: docs_2.0/manual_EN/2.introduction/1.what-is-nebula-graph.md (hunk 1 above)
Comment (oldLady344) [inline docs_2.0/manual_EN/2.introduction/1.what-is-nebula-graph.md:7]: "edges are always directed and typed." edges can be undirected.

File: docs_2.0/manual_EN/2.introduction/1.what-is-nebula-graph.md [hunk 11]
```diff
@@ -0,0 +1,69 @@
...
+
+### Intelligent question-answer system
+
+Natural languages can be transformed into knowledge graphs and stored in Nebula Graph. A question organized in a natural language can be resolved by a semantic parser in an intelligent question-answer system and re-organized. Then, possible answers to the question can be retrieved out of the knowledge graph and provided to the one who asked the question.
+
+### Social Networking
```
Comment (oldLady344) [inline docs_2.0/manual_EN/2.introduction/1.what-is-nebula-graph.md:67]: networking?

File: docs_2.0/manual_EN/2.introduction/1.what-is-nebula-graph.md [hunk 12]
```diff
@@ -0,0 +1,69 @@
...
+
+Information on people and their relationships are typical graph data. Nebula Graph can easily handle the social networking information of billions of people and trillions of relationships, and provide lightning-fast query speed for friend recommendations and job promotions in the case of massive concurrency.
```
Comment (oldLady344) [inline docs_2.0/manual_EN/2.introduction/1.what-is-nebula-graph.md:69]: provide lightning-fast query speed ==> provide lightning-fast speed query?

File: docs_2.0/manual_EN/2.introduction/1.what-is-nebula-graph.md [hunk 13]
```diff
@@ -0,0 +1,69 @@
+# What is Nebula Graph
+
+Nebula Graph is an open-source, distributed, easily scalable, and native graph database. It is capable of hosting graphs with billions of vertices and trillions of edges, and serving queries with millisecond-latency.
+
+## What is a graph database
+
+A graph database, such as Nebula Graph, is a database that specializes in storing vast graph networks and retrieving information from them. It efficiently stores data as vertices (nodes) and edges (relationships) and allows high-performance retrieval and querying of those structures. Properties can be attached to both vertices and edges. Each Vertex can have one to multiple tags.
```
Comment (oldLady344) [inline docs_2.0/manual_EN/2.introduction/1.what-is-nebula-graph.md:7]: "Each Vertex" why "vertex" is capitalized?

Comment (oldLady344) [reply docs_2.0/manual_EN/2.introduction/1.what-is-nebula-graph.md:7]: " one to multiple" ==> one or more?

**Review by whitewum — DISMISSED** [review]

File: docs_2.0/manual_EN/2.introduction/1.what-is-nebula-graph.md [hunk 14]
```diff
@@ -0,0 +1,69 @@
...
+
+### Developer friendly
+
+Nebula Graph supports clients in popular programming languages like Java, Python, C++, and Go, and more are being developed. For more information, see [Nebula Graph clients](TODO) [TODO].
+
+### Diversified ecosystem
+
+More and more native tools of Nebula Graph has been released, such as [Nebula Graph Studio](https://github.com/vesoft-inc/nebula-web-docker), [nebula-console](https://github.com/vesoft-inc/nebula-console), and [Nebula Graph Exchange](https://github.com/vesoft-inc/nebula-java/tree/master/tools/exchange). Besides, Nebula Graph has the ability to be integrated with many cutting-edge technologies, such as Spark, Flink, and Hbase, for the purpose of mutual strengthening in a world of increasing challenges and chances. For more information, see [Ecosystem development](TODO) [TODO].
```
Comment (oldLady344) [inline docs_2.0/manual_EN/2.introduction/1.what-is-nebula-graph.md:29]: have been released.

File: docs_2.0/manual_EN/2.introduction/1.what-is-nebula-graph.md [hunk 15]
```diff
@@ -0,0 +1,69 @@
...
+
+The native Nebula Graph Query Language, also known as nGQL, is a declarative, openCypher-compatible textual query language. It is easy to understand and easy to use. For more information, see [nGQL guide](https://github.com/vesoft-inc/nebula-docs/tree/master/docs_2.0/manual_EN/4.ngql-guide).
+
+### Easy data modeling and high flexibility
+
+You can easily model the connected data into Nebula Graph for your business without forcing them into a structure such as a relational table, and properties can be added, updated, and deleted freely. For more information, see [Data modeling](TODO) [TODO].
+
+### Reliable access control
+
+Nebula Graph supports strict role-based ACL and external authentication servers such as LDAP (Lightweight Directory Access Protocol) servers to enhance data security. For more information, see [Authentication and authorization](TODO) [TODO].
```
Comment (oldLady344) [inline docs_2.0/manual_EN/2.introduction/1.what-is-nebula-graph.md:41]: spell out ACL

File: docs_2.0/manual_EN/2.introduction/1.what-is-nebula-graph.md [hunk 16]
```diff
@@ -0,0 +1,69 @@
...
+
+### High scalability
+
+Nebula Graph is designed in a shared-nothing architecture and supports scaling in and out without interrupting the database service.
+
+### High popularity
+
+Nebula Graph is being used by tech leaders such as Tencent, Vivo, Meituan, JD Digits. For more information, visit the [Nebula Graph official website](https://nebula-graph.io/).
```
Comment (oldLady344) [inline docs_2.0/manual_EN/2.introduction/1.what-is-nebula-graph.md:49]: and JD Digits.

File: docs_2.0/manual_EN/2.introduction/1.what-is-nebula-graph.md [hunk 17]
```diff
@@ -0,0 +1,69 @@
...
+
+## Use cases
+
+Nebula Graph can be used to support various graph-based scenarios. To spare the time spent on pushing the kinds of data mentioned in this section into relational databases and on bothering for join queries, use Nebula Graph.
```
Comment (oldLady344) [inline docs_2.0/manual_EN/2.introduction/1.what-is-nebula-graph.md:53]: bothering with join queries?

File: docs_2.0/manual_EN/2.introduction/1.what-is-nebula-graph.md [hunk 18]
```diff
@@ -0,0 +1,69 @@
...
+
+Financial institutions have to traverse countless transactions to piece together potential crimes and understand how combinations of transactions and devices might be related to a single fraud scheme. This kind of scenario can be modeled in graphs, and with the help of Nebula Graph, fraud rings and other sophisticated scams can be easily detected.
+
+### Real-time recommendation
+
+Nebula Graph offers the ability to instantly process the real-time information produced by a visitor and make accurate recommendations on articles, videos, products, and services.
+
+### Intelligent question-answer system
+
+Natural languages can be transformed into knowledge graphs and stored in Nebula Graph. A question organized in a natural language can be resolved by a semantic parser in an intelligent question-answer system and re-organized. Then, possible answers to the question can be retrieved out of the knowledge graph and provided to the one who asked the question.
```
Comment (oldLady344) [inline docs_2.0/manual_EN/2.introduction/1.what-is-nebula-graph.md:65]: can be retrieved out of ==> is "out of" necessary or correct?

Comment (randomJoe211) [reply docs_2.0/manual_EN/2.introduction/1.what-is-nebula-graph.md:65]: Used "from" instead. Thx!

**Review by Amber1990Zhang — APPROVED** [review]
LGTM

**Review by whitewum — APPROVED** [review]

**Pull request merged**